data/core/chitchat.md

```diff
@@ -30,225 +30,134 @@
     - action_chitchat
 * signup_newsletter{"email": "maxmeier@firma.de"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": false}
-    - utter_already_subscribed
+    - utter_can_do
+    - subscribe_newsletter_form
+    - form{"name": "subscribe_newsletter_form"}
+    - form{"name": null}
     - utter_docu
     - utter_ask_feedback
 * affirm
     - utter_thumbsup
     - utter_anything_else
 
-## just newsletter (with email already) - already subscribed
+## just newsletter (with email already)
 * greet
     - action_greet_user
 * ask_weather OR ask_builder OR ask_howdoing OR ask_whoisit OR ask_whatisrasa OR ask_isbot OR ask_howold OR ask_languagesbot OR ask_restaurant OR ask_time OR ask_wherefrom OR ask_whoami OR handleinsult OR nicetomeeyou OR telljoke OR ask_whatismyname OR howwereyoubuilt
     - action_chitchat
 * signup_newsletter{"email": "maxmeier@firma.de"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": false}
-    - utter_already_subscribed
-    - utter_docu
-    - utter_ask_feedback
-
-## just newsletter +confirm - already subscribed
-* greet
-    - action_greet_user
-* ask_weather OR ask_builder OR ask_howdoing OR ask_whoisit OR ask_whatisrasa OR ask_isbot OR ask_howold OR ask_languagesbot OR ask_restaurant OR ask_time OR ask_wherefrom OR ask_whoami OR handleinsult OR nicetomeeyou OR telljoke OR ask_whatismyname OR howwereyoubuilt
-    - action_chitchat
-* signup_newsletter
-    - utter_great
-    - utter_ask_email
-* enter_data{"email": "maxmeier@firma.de"} OR enter_data{"number":"1"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": false}
-    - utter_already_subscribed
-    - utter_docu
-    - utter_ask_feedback
-* affirm
-    - utter_thumbsup
-    - utter_anything_else
-
-## just newsletter +confirm - already subscribed
-* greet
-    - action_greet_user
-* signup_newsletter
-    - utter_great
-    - utter_ask_email
-* ask_weather OR ask_builder OR ask_howdoing OR ask_whoisit OR ask_whatisrasa OR ask_isbot OR ask_howold OR ask_languagesbot OR ask_restaurant OR ask_time OR ask_wherefrom OR ask_whoami OR handleinsult OR nicetomeeyou OR telljoke OR ask_whatismyname OR howwereyoubuilt OR ask_whatspossible
-    - action_chitchat
-    - utter_ask_email
-* enter_data{"email": "maxmeier@firma.de"} OR enter_data{"number":"1"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": false}
-    - utter_already_subscribed
+    - utter_can_do
+    - subscribe_newsletter_form
+    - form{"name": "subscribe_newsletter_form"}
+    - form{"name": null}
     - utter_docu
     - utter_ask_feedback
-* affirm
```

Comment:
is there a reason we got rid of this?

Reply:
the diffs in stories are wrong when i'm deleting so much. Look at where the story starts -- I believe it's the same story that used to start in [line 127](https://github.com/RasaHQ/rasa-demo/pull/302/files#diff-86d1a6f76e93012e0fa41c3c4b379505L127), which ended at `utter_ask_feedback`

Reply:
ok i'll trust you it's fine :D